src/frontend/packages/lib_classroom/src/components/ClassroomWidgetProvider/widgets/ClassroomDescriptionWidget/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { Form, FormField, TextArea } from 'grommet';
+import { Maybe } from 'lib-common';
+import { Classroom, debounce, FoldableItem } from 'lib-components';
+import React, { useEffect, useRef, useState } from 'react';
+import toast from 'react-hot-toast';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { useUpdateClassroom } from 'data/queries';
+import { useCurrentClassroom } from 'hooks/useCurrentClassroom';
+
+const messages = defineMessages({
+  title: {
+    defaultMessage: 'Description',
+    description: 'A title for the widget.',
+    id: 'component.ClassroomDescription.title',
+  },
+  descriptionLabel: {
+    defaultMessage: 'Description',
+    description: 'Label for description in classroom creation form.',
+    id: 'component.ClassroomDescription.descriptionLabel',
+  },
+  info: {
+    defaultMessage:
+      'This widget can be used to set a description to the classroom.',
+    description: 'Helptext for the widget.',
+    id: 'component.ClassroomDescription.info',
+  },
+  welcomeTextLabel: {
+    defaultMessage: 'Welcome text',
+    description: 'Label for welcome text in classroom creation form.',
+    id: 'component.ClassroomDescription.welcomeTextLabel',
+  },
+  updateClassroomSuccess: {
+    defaultMessage: 'Classroom updated.',
+    description: 'Message when classroom update is successful.',
+    id: 'component.ClassroomDescription.updateClassroomSuccess',
+  },
+  updateClassroomFail: {
+    defaultMessage: 'Classroom not updated!',
+    description: 'Message when classroom update failed.',
+    id: 'component.ClassroomDescription.updateClassroomFail',
+  },
+});
+
+export const ClassroomDescriptionWidget = () => {
```

Comment:
I fixed this issue in an additional commit because it would involve a lot of conflicts to rename it in this PR context. I chose not to use the "widget" suffix which seemed irrelevant to me as the folder is already named "widgets"